### PR TITLE
Html refresh

### PR DESCRIPTION
**Pull request by cathyrae903**

### CODE REVIEW

File: static/styles.css [hunk 1]
```diff
@@ -399,3 +399,8 @@ form {
   -ms-transform: rotate(45deg);
   transform: rotate(45deg);
 }
+
```
Comment (GRNCheetah) [inline static/styles.css:403]: plz overload <body> with the font family since i didnt do it

**Review by GRNCheetah — CHANGES_REQUESTED** [review]
address comment

File: static/styles.css (hunk 1 above)
Comment (cathyrae903) [reply static/styles.css:403]: done

**Review by GRNCheetah — APPROVED** [review]
lgtm

**Pull request merged**